test/cnxml/index.js

```diff
@@ -29,32 +20,49 @@ global.DOMParser = dom.window.DOMParser
 global.XMLSerializer = dom.window.XMLSerializer
 global.Node = dom.window.Node
 
-const serializer = new CNXML()
+const serializer = new CNXML({documentRules: [], glossaryRules: []})
 
 function testDeserialization({ input, output }) {
-    const editor = new Editor({
-        value: serializer.deserialize(input),
-        plugins: PLUGINS,
+    const { document, glossary } = serializer.deserialize(input)
+    const editorContent = new Editor({
+        value: document,
+        plugins: CONTENT_PLUGINS,
     })
+    const editorGlossary = new Editor({
+        value: glossary,
+        plugins: GLOSSARY_PLUGINS,
+    })
+
+    if (output.content) {
+        dropKeys(editorContent.value.document).should.equal(dropKeys(output.content.document))
+    }
 
-    if (output) {
-        dropKeys(editor.value.document).should.equal(dropKeys(output.document))
+    if (output.glossary) {
+        dropKeys(editorGlossary.value.document).should.equal(dropKeys(output.glossary.document))
     }
 }
 
 function testSerialization({ input, output }) {
-    const result = serializer.serialize(dropKeys(input), { toString: false })
-        .getElementsByTagName('content')[0]
+    const contentValue = dropKeys(input.content)
+    const glossaryValue = input.glossary ? dropKeys(input.glossary) : null
+    const serialized = serializer.serialize(contentValue, glossaryValue, { toString: false })
+    const resultContent = serialized.getElementsByTagName('content')[0]
+    const resultGlossary = serialized.getElementsByTagName('glossary')[0]
 
     const referenceXml = new DOMParser().parseFromString(output, 'application/xml')
-    const reference = referenceXml.querySelector(':root > content')
+    const referenceContent = referenceXml.querySelector(':root > content')
+    const referenceGlossary = referenceXml.querySelector(':root > glossary')
 
-    if (reference == null) {
+    if (referenceContent == null) {
```

Comment:
This is a lot of additional complexity just for two tests.

Reply:
What is your suggestion?

Reply:
If you want to add more tests for glossary, set up a separate test function for them, otherwise just write those two tests without those helpers.

Reply:
I guess this will take even more time than what I've already did and it will not be simpler at all.